apps/meteor/client/views/account/preferences/AccountPreferencesPage.tsx

```diff
@@ -94,7 +94,7 @@ const AccountPreferencesPage = (): ReactElement => {
 			<PageFooter isDirty={isDirty}>
 				<ButtonGroup>
 					<Button onClick={() => reset(preferencesValues)}>{t('Cancel')}</Button>
-					<Button form={preferencesFormId} primary type='submit'>
+					<Button form={preferencesFormId} data-qa='AccountPreferencesPageSaveButton' primary type='submit'>
```

Comment:
Please do not use `data-qa` we've already talked about it internally a lot of times